attest/pcp_windows.go

```diff
@@ -404,7 +404,7 @@ func (h *winPCP) EKCerts() ([]*x509.Certificate, error) {
 
 	var out []*x509.Certificate
 	for _, der := range c {
-		cert, err := x509.ParseCertificate(der)
+		cert, err := parseCert(der)
 		if err != nil && x509.IsFatal(err) {
```

Comment:
Is `x509.IsFatal` necessary without `x509.ParseCertificate`? E.g. parseCert can return an error from `asn1.Unmarshal`. Consider moving the `x509.IsFatal` to parseCert or removing it entirely.

Reply:
Done.